torch/utils/checkpoint.py

```diff
@@ -39,59 +39,68 @@ def backward(ctx, *args):
         return (None,) + tuple(inp.grad for inp in detached_inputs)
 
 
-def checkpoint(run_function, *args):
+def checkpoint(function, *args):
```

Comment:
`function` sounds good too. I found `run_function` to be more descriptive that checkpointing will run this function directly without modifications. what do you think?

Reply:
`function` sounds better to me personally because I feel `run_function` is more of an action rather than describing an object.

Reply:
I'm open to revert this is people feel `run_function` is better!

Reply:
I too think that `function` is nicer